app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/recent/RecentRecyclerAdapter.java

```diff
@@ -2,36 +2,29 @@
 
 import android.content.Context;
 import android.support.v7.widget.RecyclerView;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ImageView;
 import android.widget.TextView;
-
 import com.bumptech.glide.Glide;
 import com.fstyle.androidtrainning.R;
 import com.fstyle.androidtrainning.model.RecentTrack;
 import com.fstyle.androidtrainning.model.RecentTracks;
-
+import com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.screen.main.mainfragments.mysong.subfragment.listsong.OnFavoriteClick. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports